packages/backend/src/functions/circuit.ts

```diff
@@ -271,6 +271,9 @@ const waitForVMCommandExecution = (ssm: SSMClient, vmInstanceId: string, command
             } catch (error: any) {
                 printLog(`Invalid command ${commandId} execution`, LogLevel.DEBUG)
 
+                const ec2 = await createEC2Client()
+                await stopEC2Instance(ec2, vmInstanceId)
```

Comment:
Perhaps adding a try-catch block here, similar to what's done below, would be a good idea.

Reply:
sure thing